### PR TITLE
chore: migrate to pnpm

### PR DESCRIPTION
**Pull request by GabsEdits**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM.

Btw wait for Mirko's review before merge since this is hosted on prem instead of GitHub pages so can't test if everything works as expected.

**Review by mirkobrombin — APPROVED** [review]
lgtm

**Pull request merged**
